components/eam/bld/build-namelist

```diff
@@ -940,7 +940,6 @@ if ($co2_cycle) {
             # This should be the same file as the one in the aircraft_specifier file.
             # This is a workaround to get this filepath into the cam.input_data_list file
             # to allow the CESM scripts to obtain all required data for a run.
```

Comment:
```suggestion

```

Reply:
The above comments aren't necessary because the unused `aircraft_co2_file` namelist entry is being purged.